compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -123,6 +126,9 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None):
         else
           s
       }
+  catch
+    case e: InvocationTargetException    => throw e
+    case e: ReflectiveOperationException => throw InvocationTargetException(e)
```

Comment:
What does this do?

Reply:
Gets caught:

https://github.com/lampepfl/dotty/blob/36353033551b6d457a58e7914bdbf21d778653f2/compiler/src/dotty/tools/repl/Rendering.scala#L150-L160

And rendered as an error:

https://github.com/lampepfl/dotty/blob/36353033551b6d457a58e7914bdbf21d778653f2/compiler/src/dotty/tools/repl/ReplDriver.scala#L372-L377

As opposed to crashing out of the REPL.

Reply:
But why is ReflectiveOperationException rewrapepd in some other exception? And isnt `case e: InvocationTargetException    => throw e` equivalent to not having a case?

Reply:
> But why is ReflectiveOperationException rewrapped in some other exception?

Because `InvocationTargetException` is what's handled upstream (in renderVal).

> And isnt `case e: InvocationTargetException => throw e` equivalent to not having a case?

No, because InvocationTargetException <: ReflectiveOperationException and I don't want to that wrapped.

Reply:
Could the code upstream be changed to handle ReflectiveOperationException instead?

Reply:
Done